src/update-content.js

```diff
@@ -0,0 +1,83 @@
+module.exports = async function (
+  owner,
+  repo,
+  old,
+  target,
+  octokit,
+  isVerbose,
+  isDryRun
+) {
+  const replacements = {
+    "README.md": [
+      {
+        from: `@${old}`,
+        to: `@${target}`,
+      },
+      {
+        from: `${owner}/${repo}.svg?branch=${old}`,
+        to: `${owner}/${repo}.svg?branch=${target}`,
+      },
+    ],
+  };
+
+  for (let path in replacements) {
+    try {
+      let file = await loadFile(owner, repo, path, octokit);
+
+      let content = file.content;
+      for (let r of replacements[path]) {
+        content = content.replace(r.from, r.to);
```

Comment:
You really want [`replaceAll`](https://v8.dev/features/string-replaceall) here, but it's not yet available in Node.js. :( Then again, maybe just doing a single replacement is good enough for now?

Reply:
Good catch, fixed!

Reply:
The new code is:

```js
        var re = new RegExp(r.from, "g");
```

It should be something like:

```js
        var re = new RegExp(escapeRegExpPattern(r.from), "g");
```

Otherwise, for example, any `.` in the input is not interpreted as a raw dot character, and there might be more severe issues.

An example `escapeRegExpPattern` implementation can be found here: https://stackoverflow.com/a/9310752/96656